### ISSUE TITLE
MSDS Components

### ISSUE THREAD

**Issue by mrmillares** [coma/msds#10]
Estoy seguro que esto esta en el roadmap. Pero un import components sería genial. Además de ver los estilos, ver los componentes en sus diferentes estados y que a su vez puedan ser modificados en el panel del DS… podría ser muy interesante para el equipo de Front.

**Comment by eduardo-syltek** [comment]
Sí, está en el roadmap pero tiene menos prioridad que el tema de integraciones.
